devices/common_patches/usb_otg/fs_v1ext.yaml

```diff
@@ -0,0 +1,105 @@
+OTG_FS_GLOBAL:
+  _copy:
+    DIEPTXF4:
+      _from: DIEPTXF1
+      addressOffset: 0x110
+    DIEPTXF5:
+      _from: DIEPTXF1
+      addressOffset: 0x114
+
+OTG_FS_HOST:
+  _copy:
+    HCCHAR8:
+      _from: HCCHAR0
+      addressOffset: 0x200
+    HCINT8:
+      _from: HCINT0
+      addressOffset: 0x208
+    HCINTMSK8:
+      _from: HCINTMSK0
+      addressOffset: 0x20C
+    HCTSIZ8:
+      _from: HCTSIZ0
+      addressOffset: 0x210
+    HCCHAR9:
+      _from: HCCHAR0
+      addressOffset: 0x220
+    HCINT9:
+      _from: HCINT0
+      addressOffset: 0x228
+    HCINTMSK9:
+      _from: HCINTMSK0
+      addressOffset: 0x22C
+    HCTSIZ9:
+      _from: HCTSIZ0
+      addressOffset: 0x230
+    HCCHAR10:
+      _from: HCCHAR0
+      addressOffset: 0x240
+    HCINT10:
+      _from: HCINT0
+      addressOffset: 0x248
+    HCINTMSK10:
+      _from: HCINTMSK0
+      addressOffset: 0x24C
+    HCTSIZ10:
+      _from: HCTSIZ0
+      addressOffset: 0x250
+    HCCHAR11:
+      _from: HCCHAR0
+      addressOffset: 0x260
+    HCINT11:
+      _from: HCINT0
+      addressOffset: 0x268
+    HCINTMSK11:
+      _from: HCINTMSK0
+      addressOffset: 0x26C
```

Comment:
Where do the offsets in this section come from?

The offset I got from the STM32F4 RM for `OTG_HS_HCINTMSKx` is `Address offset: 0x50C + 0x20 * x`, and I can't figure out how to get 0x26C from that.

Reply:
You should take into account that OTG_FS_HOST is independent peripheral with offset of 0x400

So is actually `0x400 + 0x10C + 0x20 * x`